src/app/containers/ComscoreAnalytics/Canonical/index.jsx

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable jsx-a11y/alt-text */
+import React, { useContext } from 'react';
+import { Helmet } from 'react-helmet';
+import { UserContext } from '#contexts/UserContext';
+
+const CanonicalComscoreAnalytics = () => {
+  const { cookiePolicy } = useContext(UserContext);
+  const staticAssetsPath = `${process.env.SIMORGH_PUBLIC_STATIC_ASSETS_ORIGIN}${process.env.SIMORGH_PUBLIC_STATIC_ASSETS_PATH}`;
+
+  if (cookiePolicy === '000') {
```

Comment:
This is looking good, cheers Khoa!

I've just had another check over the original issue, I think we need to do a couple of things slightly differently:

I think this needs to also execute:

```
var _comscore = _comscore || [];
_comscore.push({ c1: "2", c2: "17986528", cs_ucfr: "0" });
```
We will need to set `window._comscore` instead though, I'll find in the JIRA ticket where this snippet comes from. Note `cs_ucfr` is also set in here.

I also don't think we actually want to render null if the cookiePolicy is 000 - we can check using `personalisationEnabled` that we get from the UserContext, and then just set `csUcfr` based on that, something like:

`const csUcfr = personalisationEnabled ? "1" : "0";`